app/models/task.py

```diff
@@ -1,6 +1,13 @@
 from flask import current_app
 from app import db
+from app.models.goal import Goal
 
 
 class Task(db.Model):
-    task_id = db.Column(db.Integer, primary_key=True)
+    task_id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    title = db.Column(db.String)
+    description = db.Column(db.String)
+    completed_at = db.Column(db.DateTime, nullable=True)
+    is_complete = db.Column(db.Boolean, default=False)
```

Comment:
We don't really need a column to track whether the task is complete. We know that if `completed_at` is `None` then the task is incomplete, but if it has a value, then the task is complete. In general, we should try to avoid duplicating data that we can extrapolate from other data. Redundant data tends to become out of sync and difficult to work with.